plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/render/displayAdapter/impl/ViewportPainter.java

```diff
@@ -125,12 +125,18 @@ private void clearPane( ViewportGraphics g, int minWidth, int minHeight ) {
      * @param minHeight
      * @param minWidth
      */
-    public void paint( ViewportGraphics g, org.eclipse.swt.graphics.Image image, int minWidth,
+    public void paint( ViewportGraphics g, org.eclipse.swt.graphics.Image image, AffineTransform imageTrsf, int minWidth,
             int minHeight ) {
         processCommands(g, true);
         clearPane(g, minWidth, minHeight);
         if( image!=null ) {
```

Comment:
Wondering about formatings in the following lines. Can you please check it with formatter settings you can import from [extras/org.locationtech.udig.dev/codeformatter.xml](https://github.com/locationtech/udig-platform/blob/master/extras/org.locationtech.udig.dev/codeformatter.xml). Just re-format the changed blocks, not the whole file, so its easier to review  ;)